modules/subnet_router/ecs.tf

```diff
@@ -86,4 +86,6 @@ resource "aws_ecs_service" "tailscale" {
     security_groups  = var.security_group_ids
     subnets          = data.aws_subnets.primary.ids
   }
+
+  depends_on = [aws_efs_mount_target.primary]
```

Comment:
I realized this missing dependency today because I was bringing up a new subnet router and noticed the EFS mount **AND** the ECS service were trying to come up at the same time:

```
...
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-9d5903a121674a343"]: Still creating... [1m20s elapsed]
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-630fe20dfba21c99d"]: Still creating... [1m20s elapsed]
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-55af9bbbec7c640d6"]: Still creating... [1m20s elapsed]
module.sandbox.module.tailscale.module.subnet_router.aws_ecs_service.tailscale: Still creating... [1m20s elapsed]
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-630fe20dfba21c99d"]: Creation complete after 1m23s [id=fsmt-3de65938b7218a252]
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-55af9bbbec7c640d6"]: Creation complete after 1m23s [id=fsmt-f4574cf7058a07d06]
module.sandbox.module.tailscale.module.subnet_router.aws_efs_mount_target.primary["subnet-9d5903a121674a343"]: Creation complete after 1m23s [id=fsmt-eb520c1a57bc1826e]
module.sandbox.module.tailscale.module.subnet_router.aws_ecs_service.tailscale: Still creating... [1m30s elapsed]
module.sandbox.module.tailscale.module.subnet_router.aws_ecs_service.tailscale: Still creating... [1m40s elapsed]
...
```

and then simultaneously the ECS tasks were failing to launch with

> ResourceInitializationError: failed to invoke EFS utils commands to set up EFS volumes: stderr: Failed to resolve "fs-39ff39b5aab5e1d88.efs.us-east-2.amazonaws.com" - check that your file system ID is correct, and ensure that the VPC has an EFS mount target for this file system ID. See https://docs.aws.amazon.com/console/efs/mount-dns-name for more detail. Attempting to lookup mount target ip address using botocore. Failed to import necessary dependency botocore, please install botocore first. : unsuccessful EFS utils command execution; code: 1